src/include/distributed/version_compat.h

```diff
@@ -36,8 +36,8 @@
 #define standard_planner_compat(a, c, d) standard_planner(a, NULL, c, d)
 #define getOwnedSequencesCompat(a, b) getOwnedSequences(a)
 #define CMDTAG_SELECT_COMPAT CMDTAG_SELECT
-#define ExplainOnePlanCompat(a, b, c, d, e, f, g) ExplainOnePlan(a, b, c, d, e, f, g, \
-																 NULL)
+#define ExplainOnePlanCompat(a, b, c, d, e, f, g, h) ExplainOnePlan(a, b, c, d, e, f, g, \
+																	h)
```

Comment:
would it make more sense to break the line with a `\` directly after the define to have the replacement on one line? I believe it starts more to the left that way, which keeps enough space for the whole line to fit.

```suggestion
#define ExplainOnePlanCompat(a, b, c, d, e, f, g, h) \
    ExplainOnePlan(a, b, c, d, e, f, g, h)
```
(note, indent above might be incorrect)